courses/free-courses-en.md

```diff
@@ -769,6 +770,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://coursesity.com/course-detail/free-advanced-jenkins-in-k8s-docker-in-docker?ref=share-link) - coursesity (Google)
```

Comment:
this looks like an Udemy course why "google".
remove tracking attributes from link.